### PR TITLE
add terms of service

### PR DESCRIPTION
**Pull request by MattHJensen**
@hdoupe, could you give this a quick read to make sure everything looks good to you?

### CODE REVIEW

File: docs/TERMS.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+The parties will engage in good faith negotiations to resolve the dispute for a period of ten (10) business days after written notice of the dispute or issue is provided by one party to the other. Within such ten (10) business days, representatives from each party will engage in negotiations to resolve the dispute, and such individuals will meet in person, via videoconference or via telephone and attempt to informally resolve the dispute or issues. If those persons are unable to resolve the dispute within such ten (10) business days, then unless the parties have mutually agreed to extend the negotiation period, a party may exercise its rights available to such party under this Agreement to otherwise end the dispute.
+
+
+## Governing law
+
+Except to the extent applicable law provides otherwise, this Agreement between you and GitHub and any access to or use of the Website or the Service are governed by the federal laws of the United States of America and the laws of the State of New Jersey, without regard to conflict of law provisions. You and COMP agree to submit to the exclusive jurisdiction and venue of the courts located in the City of Atlantic Highlands, NJ.
```
Comment (hdoupe) [inline docs/TERMS.md]: Should "you and GitHub" be "you and COMP"?

**Comment by hdoupe** [comment]
@MattHJensen #37 looks good to me once we clear up the "GitHub" comment. How should this be referenced on the website? Should there be a check and a link to a page with this doc?

File: docs/TERMS.md (hunk 1 above)
Comment (MattHJensen) [reply docs/TERMS.md]: Resolved.

**Comment by MattHJensen** [comment]
Merging this. 

> How should this be referenced on the website? Should there be a check and a link to a page with this doc?

I'll address this in #35.

**Pull request merged**